src/www/ui/template/ui-clearing-view_rhs.html.twig

```diff
@@ -53,7 +53,7 @@
       {% for typeKey,typeText in clearingTypes %}
         <br/>
         <input type="radio" class="info-bullet view-license-rc-size" name="clearingTypes" value="{{ typeKey }}"
-        {% if typeKey==selectedClearingType %}checked="checked"{% endif %}/> {{ typeText }}<img src="images/info_16.png" data-toggle="tooltip" {% if typeKey==3 %} title="{{ 'Mark the file for having had a look at it, but a clearing result cannot be determined at the current time'|trans }}" {% elseif typeKey==4 %} title="{{ 'There is license relevant information, but this is not relevant information for the clearing of this file (for example, files that are not used at all and also not distributed)'|trans }}" {% else %} title="{{ 'This setting  considers the selection of licenses  shown in the table below to create a clearing result'|trans }}" {% endif %} alt="" class="info-bullet"/>
+        {% if typeKey==selectedClearingType %}checked="checked"{% endif %}/> {{ typeText }}<img src="images/info_16.png" data-toggle="tooltip" {% if typeKey==3 %} title="{{ 'Mark the file for having had a look at it, but a clearing result cannot be determined at the current time'|trans }}" {% elseif typeKey==4 %} title="{{ 'There is license relevant information, but this is not relevant information for the clearing of this file (for example, files that are not used at all and also not distributed)'|trans }}" {% elseif typeKey ==5 %} title="{{ 'There may be license relevant information but the file is a binary (.dll, .jar, etc.) and thus should not be considered as part of actual source code.'|trans }}" {% elseif typeKey ==6 %} title="{{ 'This setting considers the selection of the licenses shown in the table below to create a clearing resut.'|trans}} " {% endif %} alt="" class="info-bullet"/>
```

Comment:
Looks like you swapped the text for key 5 and 6. As key 5 is `Identified` and 6 is `Do not use`. Should be like bellow:

```twig
{% elseif typeKey==5 %} title="{{ 'This setting considers the selection of the licenses shown in the table below to create a clearing result.'|trans }}"
{% elseif typeKey==6 %} title="{{ 'There may be license relevant information but the file is a binary (.dll, .jar, etc.) and thus should not be considered as part of actual source code.'|trans}} "
```

And please check for typo.

Reply:
Well done for spotting and proposing a fix ; 
But I'm wondering about the sentence used: I don't think that we should restrict the explanation to _binary_ files. There are more diverse situations where a given file where a file should not be considered because not part of the final distribution (configuration file, generated source code, etc.)

I didn't see it discussed elsewhere, sorry if I missed something

I would propose something like:
> There may be license relevant information but this particular file should not be considered as part of actual source code (for example, excluded from distributed solution)

Reply:
As @NicolasToussaint suggested , I've made the required changes now .